### PR TITLE
fix(Slider): updated value input styling

### LINKED ISSUE #5176: Slider - input group above thumb looks odd
This is what it looks like now. There is a white background applied to the `%`, which is part of the input group - a "plain" input group "text" element

www patternfly org_v4_ (1)

Without the background

www patternfly org_v4_ (2)

With a regular input group text element

www patternfly org_v4_

With the border removed

www patternfly org_v4_ (1)

Why there is so much space there

www patternfly org_v4_

wdyt @mcarrano @mceledonia @mmenestr

### PR DESCRIPTION
**Pull request by thatblindgeye**
Closes #5176 

[Slider preview build](https://patternfly-pr-5249.surge.sh/components/slider)

Added styling to the input group text CSS to match a disabled input as well.

### CODE REVIEW

File: src/patternfly/components/Slider/slider.scss [hunk 1]
```diff
@@ -226,6 +226,7 @@
     .pf-c-input-group__text {
       position: absolute;
       left: 100%;
+      height: 100%;
```
[the remark this answers is not in the document]
Comment (mcarrano) [reply src/patternfly/components/Slider/slider.scss:229]: I agree. Centered looks better.

**Review by mcarrano — COMMENTED** [review]
I think this looks fine. But is there a reason we added extra space to the input? Looks like it has extra spacing as compared to the current implementation.

Screen Shot 2022-11-28 at 9 56 21 AM

**Comment by thatblindgeye** [comment]
> I think this looks fine. But is there a reason we added extra space to the input? Looks like it has extra spacing as compared to the current implementation (as seen below).

@mcarrano This might be some inconsistency between React and HTML examples. The HTML current implementation looks more similar to this PRs width of the input I think:

HTML slider with thumb input

React has some logic that is setting some styles to alter that width.

**Review by mcarrano — APPROVED** [review]
Thanks for the explanation @thatblindgeye . I'm good then.

**Review by mcoker — APPROVED** [review]
LGTM! Great job 👍

**Pull request merged**